CMakeLists.txt

```diff
@@ -396,11 +396,11 @@ endif ()
 # ----------------------------------------------------------------------------
 # installation directories
 if (OS_WINDOWS)
-  set (RUNTIME_INSTALL_DIR Bin)
-  set (LIBRARY_INSTALL_DIR Lib)
-  set (INCLUDE_INSTALL_DIR Include)
-  set (CONFIG_INSTALL_DIR  CMake)
-  set (PKGCONFIG_INSTALL_DIR)
+  set (RUNTIME_INSTALL_DIR "bin")
```

Comment:
Sorry, I didn't get back yet. The reason for capital letters is because it's Windows...

1. Filesystem is generally case insensitive.
2. Browse through your "C:\Program Files" folder and you'd see plenty of folders starting with upper case letters instead of the Unix style naming with all lowercase. A more specific example would be the Python (and I assume Perl as well) installation on Windows. Most Windows users I would claim use a GUI (e.g. Windows Explorer) to browse their files rather than the command line and therefore there is no difference in convenience regarding typing a folder name upper or lower case.

Anyway, that's just been the motivation as I believe it fits more into the common "naming convention" on Windows (if there is one, judging from examples only).

I don't mind having the same folder names for all platforms.